src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileFrom = new File(fromFileName);
+        File fileTo = new File(toFileName);
+        try {
```

Comment:
Don't need to create files
Also, it's better to use `try (BufferedReader reader = new BufferedReader(new FileReader(sourceFile)))`
for getting data from file